lib/account/pages/help_page.dart

```diff
@@ -37,7 +87,8 @@ class _HelpPageState extends State<HelpPage> {
                 ),
                 SubmitButton(
                   text: S.of(context).pageHelpContactUs,
-                  onPressed: () => print("Contact TODO"),
+                  onPressed: () => launchUrlString(
+                      'mailto:motismitfahrapp@gmail.com?subject=${S.of(context).pageHelpEmailSubject(SupabaseManager.getCurrentProfile()!.id!)}'),
```

Comment:
comma missing

Reply:
Where? This is just a long string

Reply:
At the end? That's the bracket from `launchUrlString`